### PR TITLE
[libc++][CI] Tests LLDB libc++ data formatters.

### PR DESCRIPTION
**Pull request by mordante**
This enables testing of the LLDB libc++ specific data formatters.
This is enabled in the bootstrap build since building LLDB requires Clang and this
is quite expensive. Adding this test changes the build time from 31 to 34 minutes.

### CODE REVIEW

File: lldb/packages/Python/lldbsuite/test/lldbtest.py [hunk 1]
```diff
@@ -751,6 +751,8 @@ def setUpCommands(cls):
             "settings set symbols.enable-external-lookup false",
             # Inherit the TCC permissions from the inferior's parent.
             "settings set target.inherit-tcc true",
+            # Based on https://discourse.llvm.org/t/running-lldb-in-a-container/76801/4
+            "settings set target.disable-aslr false",
```
Comment (mordante) [inline lldb/packages/Python/lldbsuite/test/lldbtest.py:755]: The reason for change is in the Discourse link. This probably needs to be done in a separate commit.

**Comment by Michael137** [comment]
Awesome, thanks for doing this!!

**Comment by jasonmolenda** [comment]
The lldb change part of this PR looks good to me.

**Comment by medismailben** [comment]
LGTM! Thanks!

**Review by ldionne — APPROVED** [review]
LGTM, thanks for picking this up!

**Comment by adrian-prantl** [comment]
@mordante This broke the green dragon bot

https://green.lab.llvm.org/job/llvm.org/view/LLDB/job/lldb-cmake/1069/

This change seems to break the `frame diagnose` test:

+            # Based on https://discourse.llvm.org/t/running-lldb-in-a-container/76801/4
+            "settings set target.disable-aslr false",

**Comment by adrian-prantl** [comment]
I worked around the problem in 466017c8dab74f66ce513c8752f0c1dcd16a8a63.

**Comment by mordante** [comment]
Thanks for the fix!
